roles/common/tasks/main.yml

```diff
@@ -39,6 +69,10 @@
   when: env != 'development' and darwin_without_passlib | default(false)
   run_once: true
 
+- name: Verify role requirements are fulfilled
+  include_tasks: "{{ item }}"
+  with_items: "{{ (playbook_dir + '/roles/*/tasks/validate.yml') | fileglob }}"
+
```

Comment:
I think this `include_tasks` is no longer needed, right? (there's no more `validate.yml`)

Reply:
Removed